man/roxygen/meta.R

```diff
@@ -0,0 +1,17 @@
+list(
+  rd_family_title = list(
+    adjacency = "Adjacency matrices",
+    attributes = "Vertex, edge and graph attributes",
+    bipartite = "Bipartite graphs",
+    centrality = "Centrality measures",
+    coloring = "Graph coloring",
+    community = "Community detection",
+    components = "Connected components",
+    epi = "Processes on graphs",
+    eulerian = "Graph cycles",
```

Comment:
or should it remain "eulerian"?

Reply:
No, the idea is that as igraph gets more cycle-related functions, they will all go into this category.

E.g. C/igraph 0.10 has cycle basis functions. Like here: https://igraph.org/c/html/latest/igraph-Cycles.html

So I would also rename the identifiers, not just the titles.  `eulerian` -> `cycles`, `epi` -> `processes`, etc.